### PR TITLE
mds: permit implicit allow in mds caps

### PR DESCRIPTION
**Pull request by batrick**
Fixes: https://tracker.ceph.com/issues/22839

Signed-off-by: Patrick Donnelly <pdonnell@redhat.com>

### CODE REVIEW

**Comment by dillaman** [comment]
FWIW - the code change linked in the the associated tracker just made the “allow” prefix optional for profiles but still required it for explicit caps.

**Comment by batrick** [comment]
Hmm, client caps don't have a profile analog. @jcsp, I don't think we need this?

File: src/test/mds/TestMDSAuthCaps.cc [hunk 1]
```diff
@@ -234,6 +242,8 @@ TEST(MDSAuthCaps, OutputParsed) {
     const char *output;
   };
   CapsTest test_values[] = {
+    {"",
+     "MDSAuthCaps[allow rw]"},
     {"allow",
```
Comment (ukernel) [inline src/test/mds/TestMDSAuthCaps.cc]: does this mean "empty string implies rw"?

Comment (batrick) [reply src/test/mds/TestMDSAuthCaps.cc]: Yes, but I don't like it.

**Comment by jcsp** [comment]
Ah, never mind!

**Pull request closed without merging**